capstone-frontend/src/components/MyClubs.jsx

```diff
@@ -69,23 +86,35 @@ class MyClubs extends Component {
       clubArray.push(<ClubGridItem key={c.id} club={c} />);
     });
     return (
-      <div className="page-container">
+      <div>
         <Row>
-          <Col xs={12} className="title"> My Clubs </Col>
+          <Col>
+            <h2 className="ml-2">My Clubs</h2>
```

Comment:
nit: This render function has a large mix of single and double quotes. I would standardize to single quotes

Reply:
Ok